gnat2goto/driver/goto_utils.ads

```diff
@@ -7,6 +7,8 @@ with Uintp;                 use Uintp;
 
 package GOTO_Utils is
 
+   type Irep_Array is array (Integer range <>) of Irep;
```

Comment:
We already have `Irep_Array` in `arrays.ads`

Reply:
Indeed. Should I move it? It seems more like a utility rather than an array-specific thing.